packages/web-components/src/divider/divider.docs.mdx

```diff
@@ -0,0 +1,64 @@
+import { Meta, Story } from '@storybook/addon-docs';
+import Divider from './divider.stories.ts';
+
+<Meta title="UX Web Components/Divider" parameters={{ status: { type: 'experimental' } }} />
+
+# Divider
+
+A divider groups sections of content to create visual rhythm and hierarchy. Use dividers along with spacing and headers to organize content in your layout.
+
+<Story id="web-components-divider--divider" />
+
+## Attributes
+
+| attribute           | type                                         | description                                                                                                                                              |
+| ------------------- | -------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| dividerAlignContent | "center" \| "start" \| "end"                 | Fluent v9. Determines the alignment of the content within the divider. DividerAlignContent.center is default.                                            |
+| dividerAppearance   | "strong" \| "brand" \| "subtle" \| "default" | Fluent v9. A divider can have one of the preset appearances. When not specified, the divider has its default: DividerAppearance.default.                 |
+| inset               | boolean                                      | Fluent v9. Divider layout is false (block) by default for strict distinctions between items, or inset for closer relationships with neighboring content. |
+| role                | "separator" \| "presentation"                | Inherited from FASTDivider. Aria role for the divider. Separator is default.                                                                             |
+| orientation         | "horizontal" \| "vertical"                   | Inherited from FASTDivider. Layout can be horizontal (default) or vertical. Adds aria-orientation to component.                                          |
+
+## Accessibility
+
+The Divider component has a role attribute which is set either to separator or presentation.
+
+## Usage
+
+Default configuration
+
+```html
+<!-- in your project.html -->
+
+<dcs-divider></dcs-divider>
+```
+
+Inset
+
+```html
+<!-- in your project.html -->
+
+<dcs-divider inset="true"></dcs-divider>
+```
+
+Block with slotted text
+
+```html
+<!-- in your project.html -->
+
+<dcs-divider>
+  <dcs-text slot="content" size="48">Greetings</dcs-text>
+</dcs-divider>
+```
+
+Block with slotted SVG
+
+```html
+<!-- in your project.html -->
+
+<dcs-divider>
+  <span slot="content">
```

Comment:
Not sure what this is for, but I assume this is just the default slot so this likely isn't needed.

Reply:
I was writing the spec/docs without yet building out the component.